### DISCUSSION TITLE
Is there a way to disable the 'always open new browser tab' feature when using "translate with"?

### DISCUSSION THREAD

**Discussion by FiiQi** [sumatrapdfreader/sumatrapdf#3916] [Q&A]
I search for selections like the image below.
----
![image_2023-11-27_160748060](https://github.com/sumatrapdfreader/sumatrapdf/assets/29916012/b3792842-7a0a-43a2-9e1e-332c513b279e)

 I will have trouble if I search for a longer time
----
![image_2023-11-27_160954572](https://github.com/sumatrapdfreader/sumatrapdf/assets/29916012/7c1792ae-7ff2-49cc-a526-3227b7040fc8)
---
Thank you!

**Comment by GitHubRulesOK** [comment]
probably not as that is a browser function

**Reply by kjk** [reply]
Unfortunately we don't have control over that.
